salt/utils/aws.py

```diff
@@ -547,12 +538,7 @@ def query(
         items = root
 
     if setname:
-        if sys.version_info < (2, 7):
-            children_len = len(root.getchildren())
-        else:
-            children_len = len(root)
-
-        for item in range(0, children_len):
+        for item in range(0, len(root)):
```

Comment:
Apply the same enumerate logic I commented above.

Reply:
Thanks, @s0undt3ch -- I will fix this and the other one. I won't be able to get to this for a few days.

Reply:
@s0undt3ch, I've just rebased, fixed according to your notes (I think), and force-pushed. Thanks for your patience.